### ISSUE TITLE
HoloLens 2 - Lost Tracking message using XRSDK subsystem

### ISSUE THREAD

**Issue by Kialarr** [microsoft/MixedReality-WorldLockingTools-Unity#300]
Hi, 
I'm trying to build the WorldLockingBasic scene from Examples to HoloLens 2, setting Anchor subsystem to XRSDK and applying the WLT HL2 MRTK profile, but for some reason I can't get it to work on the device - I'm getting the Lost Tracking message, no further explanation. Do you know if I could have missed something in the configuration? I'm using Unity 2021.3 but also tried it on 2020 LTS with same result.

**Comment by mrvnbrk** [comment]
Hi,
I have the same problem. Have you been able to solve it in the meantime? I will let you know if I find a solution.

**Comment by Kialarr** [comment]
No, I just switched to AR Foundation and its XR Anchor Store, I won't bother to work with a solution where I can't even get the examples to run properly.

**Comment by mrvnbrk** [comment]
Do you have documentation for this solution? I am still a beginner and need support.

**Comment by BookSadprasid** [comment]
@fast-slow-still I am also facing the same issue using the **WorldLockedPhysicsSample**. 

Here are the details of my setup.

- **Unity Version**: `2020.3.37f1 LTS`

**Custom Packages:**
- **Mixed Reality OpenXR Plugin**: `1.4.3`
- **Mixed Reality Toolkit Foundation**: `2.8.2`
- **Mixed Reality Standard Assets**: `2.8.2`
- **WLT Core**: `1.5.9`
- **WLT Samples**: `1.5.9`

**Unity Technologies Packages**:
- **OpenXR Plugin**: `1.4.2` (I wanted to note this one in case it conflicts with the Mixed Reality one)
- others but let me know if you need this.

The only changes to the base sample are: 
- Changing the **MixedRealityToolkit** profile from `DefaultMixedRealityToolkitConfigurationProfile` to `DefaultHoloLens2ConfigurationProfile`
- Changing the **World Locking Context** script setting in the Unity editor to **Use Default** for the **Automation Settings**
- Changing the **World Locking Context** script settings in the Unity editor to use the `XRSDK` **Anchor Subsystem** since that is the default selected when using the *Configure Scene* tool.

Deployment-wise, I just build it using the MRTK/Unity recommended build settings and deploy it remotely via Visual Studio 2022.

This is a screenshot showing the **Loss Tracking** error message a **Current Fragment: Invalid**. Due to this error, no Anchors or edges are being instantiated. 

![20220730_143420_HoloLens](https://user-images.githubusercontent.com/42843916/181935187-d72ae1bc-6e1e-4c96-88ac-05731b82c464.jpg)

**Comment by KoltonG** [comment]
+1 for @BookSadprasid's issue. I am facing the exact same problem.

I've also tried to change the anchor subsystem to ARFoundation with the same issue of "Lost Tracking".

This is unfortunately blocking my project from using WLT since both XRSDK and ARFoundation are not working.

**Comment by johnluksas** [comment]
Hello All

I was facing the same issue on the examples but was able to fix it by rolling back the Mixed Reality OpenXR Plugin to `1.4.1`. I have not tried Mixed Reality OpenXR Plugin `1.4.2` so it is possible that version may work.

**Old Config (did not work for me)**
Unity: `2020.3.34f1`
Mixed Reality OpenXR Plugin: `1.4.3`
OpenXR Plugin: `1.4.2`
XR Plugin Management: `4.2.1`
Anchor Sub-system: `XRSDK`

**New Config (worked for me)**
Unity: `2020.3.34f1`
Mixed Reality OpenXR Plugin: `1.4.1`
OpenXR Plugin: `1.4.2`
XR Plugin Management: `4.2.1`
Anchor Sub-system: `XRSDK`

@BookSadprasid @KoltonG Let me know if that works for you so I know that was the actual issue and not another thing I happened to change.

The easiest way to change the Mixed Reality OpenXR Plugin version is in the dropdown on the Mixed Reality Feature Tool

<img width="514" alt="image" src="https://user-images.githubusercontent.com/13530231/182438887-b3094b92-5e78-42b1-b0a3-342dadb79dd9.png">

**Comment by mrvnbrk** [comment]
@crusher1013  That's it! Thank you!

**Comment by BookSadprasid** [comment]
@crusher1013 thank you so much, that was it! I tried with **Mixed Reality OpenXR Plugin**: `1.4.2` and got the "Lost Tracking" error so for now I will stick with  **Mixed Reality OpenXR Plugin**: `1.4.1`
